cypress/e2e/inventory/holdings/save-empty-statistical-code-on-holdings-page.cy.js

```diff
@@ -0,0 +1,136 @@
+import { including } from 'bigtest';
+import { randomFourDigitNumber } from '../../../support/utils/stringTools';
+import { DevTeams, Permissions, TestTypes } from '../../../support/dictionary';
+import InventoryInstances from '../../../support/fragments/inventory/inventoryInstances';
+import Users from '../../../support/fragments/users/users';
+import InventorySearchAndFilter from '../../../support/fragments/inventory/inventorySearchAndFilter';
+import InventoryInstance from '../../../support/fragments/inventory/inventoryInstance';
+import HoldingsRecordEdit from '../../../support/fragments/inventory/holdingsRecordEdit';
+import TopMenu from '../../../support/fragments/topMenu';
+import HoldingsRecordView from '../../../support/fragments/inventory/holdingsRecordView';
+import { LOCATION_NAMES } from '../../../support/constants';
+
+const testData = {
+  user: {},
+  item: {
+    instanceName: `Inventory Instance ${randomFourDigitNumber()}`,
+    itemBarcode: randomFourDigitNumber(),
+  },
+  statisticalCode: 'Book, print (books)',
+  secondStatisticalCode: 'Books, electronic (ebooks)',
+  errorMessage: 'Please select to continue',
+  calloutMessage: 'has been successfully saved.',
+};
+
+describe('Holdings', () => {
+  before('Create test data', () => {
+    cy.createTempUser([Permissions.inventoryAll.gui]).then((userProperties) => {
+      testData.user = userProperties;
+      cy.getAdminToken()
+        .then(() => {
+          cy.getInstanceTypes({ limit: 1 });
+          cy.getInstanceIdentifierTypes({ limit: 1 });
+        })
+        .then(() => {
+          cy.createInstance({
```

Comment:
better to use the new approach for creating instances
example cypress/e2e/loans/multiple-loans.cy.js